src/lib/isMobilePhone.js

```diff
@@ -28,6 +28,7 @@ const phones = {
   'en-IE': /^(\+?353|0)8[356789]\d{7}$/,
   'en-IN': /^(\+?91|0)?[6789]\d{9}$/,
   'en-KE': /^(\+?254|0)(7|1)\d{8}$/,
+  'en-MO': /^(\+?853\-?)?6\d{3}\-?\d{4}$/,
```

Comment:
Good work adding this new feature. 
I just wanted to leave some comments, 

1. Add tests for the same (Valid and invalid cases). 
2. Update the README file to help people that the new mobile validation is available. 

In case of any clarification, feel free to ask.